_opensearch/install/plugins.md

```diff
@@ -63,27 +63,27 @@ bin/opensearch-plugin list
   </thead>
   <tbody>
     <tr>
-    <td>1.3.2</td>
+    <td>2.1.0</td>
     <td>
-      <pre>opensearch-alerting                  1.3.2.0
-opensearch-anomaly-detection         1.3.2.0
-opensearch-asynchronous-search       1.3.2.0
-opensearch-cross-cluster-replication 1.3.2.0
-opensearch-index-management          1.3.2.0
-opensearch-job-scheduler             1.3.2.0
-opensearch-knn                       1.3.2.0
-opensearch-ml                        1.3.2.0
-opensearch-observability             1.3.2.0
-opensearch-performance-analyzer      1.3.2.0
-opensearch-reports-scheduler         1.3.2.0
-opensearch-security                  1.3.2.0
-opensearch-sql                       1.3.2.0
+      <pre>opensearch-alerting                  2.1.0.0
+opensearch-anomaly-detection         2.1.0.0
+opensearch-asynchronous-search       2.1.0.0
+opensearch-cross-cluster-replication 2.1.0.0
+opensearch-index-management          2.1.0.0
+opensearch-job-scheduler             2.1.0.0
+opensearch-knn                       2.1.0.0
+opensearch-ml                        2.1.0.0
+opensearch-observability             2.1.0.0
+opensearch-performance-analyzer      2.1.0.0
+opensearch-reports-scheduler         2.1.0.0
+opensearch-security                  2.1.0.0
+opensearch-sql                       2.1.0.0
```

Comment:
This is the 1.3 branch; should it even acknowledge that a v2 exists?

Reply:
Removed v2